### ISSUE TITLE
How to store references to the views? (ViewHolder support?)

### ISSUE THREAD

**Issue by travisdachi** [Judrummer/JxAdapter#1]
In the example you use "kotlin-android-extensions" extension properties in your bind function, right?
I understand you get the caching function only with Activity/Fragment not with some arbitrary view.
That means you basically call findViewById every time in onBind() which we're not supposed to do.
How should I approach this?

**Comment by Judrummer** [comment]
ahh , I forgot that kotlin-android-extensions not keep cache for normal-view. 
I will try to think some good solution to solve.
If you have some good idea, you can suggest me or open pull request.

**Comment by Judrummer** [comment]
@travissuban  
https://github.com/yanex/KEEP/blob/9e33134d33ef242640ab476cce265a2b3a83544d/proposals/android-extensions-entity-caching.md
I found this in KEEP pull request.

**Comment by travisdachi** [comment]
That's a good news for all of us. But even if that feature is available. We still need our own ViewHolder, not the JxViewHolder, right?

**Comment by travisdachi** [comment]
Oh! I figured it out. We can always use the good old ListView's view holder pattern where we store the view holder in `view.tag`

**Comment by Judrummer** [comment]
@travissuban  
If I change lambda block to return viewholder as this way ,you think it's better?

`
class JxViewHolder<T : Any>(val itemLayoutId: Int, val itemType: Class<T>, val bindHolder: RecyclerView.ViewHolder.(position: Int, item: T) -> Unit)
`
```

    JxViewHolder<HeaderViewData>(R.layout.item_test_jx_header) { position, item ->
                    //viewholder scope  this=viewHolder
                    itemView.tvTestItemHeader  = item.name
                }
```

**Comment by travisdachi** [comment]
That's perfect! Now all we have to do is wait for the view caching.

**Comment by Judrummer** [comment]
@travissuban  okay, thanks for your suggestion :)

**Comment by Judrummer** [comment]
I have fixed it in 0.2.0 release already.

**Comment by travisdachi** [comment]
Cool! Thanks. I'll close this issue.
